applications/ecallmgr/src/ecallmgr_registrar.erl

```diff
@@ -923,8 +922,13 @@ query_authn(#registration{username=Username
             AccountId = wh_json:get_value(<<"Account-ID">>, CCVs),
             AccountDb = wh_util:format_account_id(AccountId, 'encoded'),
             AuthorizingId = wh_json:get_value(<<"Authorizing-ID">>, CCVs),
+            OwnerIdProp = case wh_json:get_value([<<"Custom-Channel-Vars">>, <<"Owner-ID">>], JObj) of
+                              'undefined' -> [];
+                              OwnerId -> [{'db', OwnerId}]
```

Comment:
Did you test this, surely you would need to bind the cache to user document updates ;)